### PR TITLE
Tycho r2.0.1+

### PR DESCRIPTION
**Pull request by yhetti**

### CODE REVIEW

**Review by martinling — CHANGES_REQUESTED** [review]
With the new footprint for U4, pins 1, 2, 4, 5 and 9 now have tracks running onto them that are wider than the pads. These tracks should be reduced so that the fabricated pads consistently have the width defined in the footprint.

image

For the same reasons, I think it would be better to remove the GND polygon around pin 3, and run a track into this pin instead. When trying to rework this part, the extra heat capacity of the GND net was a significant issue.

On Cynthion we do have a polygon around pin 3, but we also have a polygon around pin 8 which serves to balance it out on the opposite corner. On Tycho we don't have this option, as pin 8 is in use to disable the switch, rather than being permanently grounded.

On the User.Drawings layer, the 92mm dimension on the right is no longer aligned with the holes. Its value is correct but it has been shifted upward slightly. The new dimensions look OK. Should there perhaps be one for the 24V socket too?

I think that the TPS55288 footprint is still not quite correct. The top edges of pads 1 and 19 do not line up with the top edges of pads 20-23.

File: tycho/tycho.kicad_pro [hunk 1]
```diff
@@ -648,7 +648,7 @@
     ]
   ],
   "text_variables": {
-    "DATE": "2024-03-07",
-    "VERSION": "r2.0.1"
+    "DATE": "2024-04-09",
+    "VERSION": "r2.0.1+"
```
Comment (mossmann) [inline tycho/tycho.kicad_pro:652]: This is fine, but in the future please try to remember to update the version with a "+" before making any other changes.

File: tycho/tycho.kicad_pcb [hunk 2]
```diff
@@ -4417,7 +4417,7 @@
         (effects (font (size 0.7 0.7) (thickness 0.15)))
       (tstamp 9c904c40-d6ca-41d1-8f30-fe11b7be348a)
     )
-    (fp_text value "42.2k" (at 2.1 0 -180) (layer "F.Fab")
+    (fp_text value "22k" (at 2.1 0 -180) (layer "F.Fab")
```
Comment (mossmann) [inline tycho/tycho.kicad_pcb:4420]: The change to the R65 value is layout/schematic parity fix for r2.0.1. This is fine, but in the future try to make such changes first before updating the version to "+".

File: tycho/tycho.kicad_pcb [hunk 3]
```diff
@@ -8215,83 +8215,83 @@
...
-      (net 162 "/D_TEST_PLUS") (pinfunction "2D+") (pintype "passive") (tstamp 3c7fc71a-5672-4de1-9502-a2ce43c6b2f5))
-    (pad "7" smd rect (at 0.5685 -0.2 180) (size 0.563 0.225) (layers "F.Cu" "F.Paste" "F.Mask")
-      (net 163 "/D_TEST_MINUS") (pinfunction "2D-") (pintype "passive") (tstamp dfe8f49d-5cad-4a93-b945-b323b38e1e14))
-    (pad "8" smd rect (at 0.4 -0.7685 180) (size 0.225 0.563) (layers "F.Cu" "F.Paste" "F.Mask")
-      (net 23 "/D_OE!_1") (pinfunction "OE!") (pintype "input") (tstamp d562e26c-74f4-48da-851e-1c93ab0bcaf6))
-    (pad "9" smd rect (at 0 -0.7685 180) (size 0.225 0.563) (layers "F.Cu" "F.Paste" "F.Mask")
-      (net 2 "VCC") (pinfunction "VCC") (pintype "power_in") (tstamp b1d4cbe1-7ff0-4d00-8ed3-fd844db4a8bb))
-    (pad "10" smd rect (at -0.4 -0.7685 180) (size 0.225 0.563) (layers "F.Cu" "F.Paste" "F.Mask")
-      (net 22 "/D_S_1") (pinfunction "S") (pintype "input") (tstamp f720bc90-c600-4461-888c-a9843fa094a7))
+    (fp_text value "PI3USB102G" (at 0 2 180) (layer "F.Fab")
```
Comment (mossmann) [inline tycho/tycho.kicad_pcb]: Please hide the U4 Value on F.Fab.

File: tycho/tycho.kicad_pcb [hunk 4]
```diff
@@ -8215,83 +8215,83 @@
...
     (path "/adaa3f42-a370-4a4c-b378-51d1ea5352f8/74af83db-188e-478d-946e-80257d1c6ddb")
     (attr smd)
-    (fp_text reference "U4" (at -0.2 1.7 180) (layer "F.SilkS")
-        (effects (font (size 0.7 0.7) (thickness 0.15)))
-      (tstamp e442a56f-2c9c-4500-8543-a855abb102c6)
-    )
-    (fp_text value "~" (at 1.5 0 90) (layer "F.Fab") hide
-        (effects (font (size 0.5 0.5) (thickness 0.1)))
-      (tstamp df6c8133-8cc5-42e8-b0c1-8d89c794ad9f)
+    (fp_text reference "U4" (at 0 -2 180) (layer "F.SilkS")
```
Comment (mossmann) [inline tycho/tycho.kicad_pcb]: Please fix the "U4" silkscreen conflict on F.SilkS.

**Review by mossmann — CHANGES_REQUESTED** [review]
The U6 footprint is much improved, but I would also like to see silkscreen, fab, and courtyard layers added to it. I've requested some other small changes as well.

**Comment by mossmann** [comment]
This isn't specific to this PR in particular, but I would like .kicad_prl files removed from the repo and added to .gitignore.

**Review by mossmann — CHANGES_REQUESTED** [review]
The U6 footprint updates look good.

Only one end of the 92 mm dimension was corrected, making the dimension value incorrect.

The trace widths escaping from U4 are generally improved, but I would like to see pins 1 and 2 routed nicely like pins 4 and 5, avoiding the excess pin 1 pad width.

Fix the clearance violations introduced by the changes to `D_SW1+` and `D_SW1-` routing. These can be fixed by refilling zones.

Always run DRC and refill zones before creating or updating a pull request.

Remove the zone around U4 pin 3. A commit message says this was done, but the zone is still there.

Restore the `HOST_VBUS` zone that became unfilled. This can be fixed by refilling zones.

Add a dimension for the 24 V socket (J8) as @martinling suggested.

**Comment by martinling** [comment]
The dedicated polygon around U4 pin 3 was removed, but the main GND polygon for the top layer still fills some of the area around this pin. To get a clean track running into that pin it's necessary to add a keepout area.

image
image

I have this change in 633c8e664edbbf24113529f95da32f2e54311757 if you want it.

**Review by mossmann — CHANGES_REQUESTED** [review]
The updates look good, but please add a keepout to U4 pin 3 as @martinling suggested.

**Review by mossmann — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**